tf-modules/cumulus/ecs_cluster_instance_autoscaling_cf_template.yml.tmpl

```diff
@@ -23,13 +23,19 @@ Resources:
           --==BOUNDARY==
           Content-Type: text/cloud-boothook; charset="us-ascii"
 
+%{ if docker_storage_driver == "devicemapper" ~}
+          vgcreate docker /dev/xvdcz
+
+          lvcreate -n docker-data -L${docker_volume_create_size}G docker
+
           sed -i '/^\s*DOCKER_STORAGE_OPTIONS=/d' /etc/sysconfig/docker-storage
-          echo 'DOCKER_STORAGE_OPTIONS="--storage-driver ${docker_storage_driver}"' >> /etc/sysconfig/docker-storage
+          echo 'DOCKER_STORAGE_OPTIONS="--storage-driver devicemapper --storage-opt dm.datadev=/dev/docker/docker-data"' >> /etc/sysconfig/docker-storage
```

Comment:
I don't think this --storage-opt flag is present in `app/cloudformation.template/yml`, you may need to update that as well.

Reply:
And check for any additional discrepancies between that script and this one.

Reply:
@nemreid it's not super straightforward in kes and there are workarounds anyway, so opted just to really fix this in TF

Reply:
The issue is you can't allocate for the volume_size, you need to do volume_size + 1, which is super easy in TF and not really worth the trouble in kes

Reply:
OK, makes sense. Thanks.